packages/amazonq/package.json

```diff
@@ -13,6 +13,7 @@
         "type": "git",
         "url": "https://github.com/aws/aws-toolkit-vscode"
     },
+    "homepage": "https://github.com/aws/aws-toolkit-vscode",
```

Comment:
Is this change still necessary since we are no longer running the `setup-test` command?